README.md

```diff
@@ -95,9 +95,10 @@ Note: The steps above are for a debug, permissive mode build and includes deprec
 
 ## Send commands, receive telemetry
 
-The cFS-GroundSystem tool can be used to send commands and receive telemetry (see https://github.com/nasa/cFS-GroundSystem/tree/master/Guide-GroundSystem.txt, the Guide-GroundSystem.txt).  Note it depends on PyQt5 and PyZMQ:
+The cFS-GroundSystem tool can be used to send commands and receive telemetry.  For details on using and setting up the Ground System, see the Guide-GroundSystem.txt: https://github.com/nasa/cFS-GroundSystem/tree/master/Guide-GroundSystem.txt.  Note it depends on PyQt5 and PyZMQ:
```

Comment:
Use markdown link syntax https://www.markdownguide.org/basic-syntax#links for these files